smf/nexus/multi-sled/config-partial.toml

```diff
@@ -24,6 +24,13 @@ if_exists = "append"
 # [schema]
 # schema_dir = "/var/nexus/schema/crdb"
 
+[tunables]
+max_vpc_ipv4_subnet_prefix = 26
+
+# Reserve space for non-Crucible storage
+# See oxidecomputer/omicron#7875 for the 250G determination.
+control_plane_storage_buffer_gb = 250
+
```

Comment:
The bigger question I have is: does a4x2 not use this file?  or: have you tested that this does the right thing on a4x2?